app/src/main/java/me/saket/dank/ui/accountmanager/AccountManagerActivity.java

```diff
@@ -0,0 +1,325 @@
+import me.saket.dank.utils.ItemTouchHelperDragAndDropCallback;
+import me.saket.dank.utils.RxDiffUtil;
+import me.saket.dank.utils.itemanimators.SlideUpAlphaAnimator;
+import me.saket.dank.widgets.swipe.RecyclerSwipeListener;
+
+@DeepLink(AccountManagerActivity.DEEP_LINK)
+@RequiresApi(Build.VERSION_CODES.N_MR1)
+public class AccountManagerActivity extends DankActivity {
+  public static final String DEEP_LINK = "dank://accountManager";
+
+  private final int ACTION_DELETE = 1;
+  private final int ACTION_LOGOUT = 2;
+  private int ACTION = 0;
+
+  private AccountManager selectedAccount = null;
+
+  @BindView(R.id.account_manager_root) ViewGroup rootViewGroup;
+  @BindView(R.id.account_manager_accounts_recyclerview) RecyclerView accountsRecyclerView;
+  @BindView(R.id.account_manager_logout) Button logoutButton;
+  @BindView(R.id.account_progressbar) View loadingProgessbar;
+
+  @Inject Lazy<UserSessionRepository> userSessionRepository;
+  @Inject Lazy<SubscriptionRepository> subscriptionRepository;
+  @Inject Lazy<AccountManagerRepository> accountManagerRepository;
+  @Inject Lazy<AccountManagerAdapter> accountManagerAdapter;
+  @Inject Lazy<ErrorResolver> errorResolver;
+
+  private Disposable confirmTimer = Disposables.disposed();
+  private Disposable timerDisposable = Disposables.empty();
+
+  @CheckResult
+  public static Intent intent(Context context) {
+    return new Intent(context, AccountManagerActivity.class);
+  }
+
+  @Override
+  protected void onCreate(@Nullable Bundle savedInstanceState) {
+    Dank.dependencyInjector().inject(this);
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_account_manager);
+    ButterKnife.bind(this);
+  }
+
+  @Override
+  protected void onPostCreate(@Nullable Bundle savedState) {
+    super.onPostCreate(savedState);
+
+    setupUserList();
+  }
+
+  @Override
+  protected void onSaveInstanceState(Bundle outState) {
+    super.onSaveInstanceState(outState);
+  }
+
+  @Override
+  @OnClick(R.id.account_manager_done)
+  public void finish() {
+    timerDisposable.dispose();
+    super.finish();
+  }
+
+  @OnClick(R.id.account_manager_logout)
+  public void logout() {
+    confirmAction(ACTION_LOGOUT);
+  }
+
+  private Completable queueToDelete(AccountManager account){
+    this.selectedAccount = account;
+    return this.confirmAction(ACTION_DELETE);
+  }
+
+  private Completable confirmAction(int action) {
+    if (confirmTimer.isDisposed()) {
+      ACTION = action;
+      int confirmText = ACTION_LOGOUT == ACTION ? R.string.userprofile_confirm_logout : R.string.userprofile_confirm_delete;
+
+      runOnUiThread(() -> {
+        // Stuff that updates the UI
+        logoutButton.setText(confirmText);
+        logoutButton.setVisibility(View.VISIBLE);
+      });
+
+
+      confirmTimer = Observable.timer(5, TimeUnit.SECONDS)
+          .compose(applySchedulers())
+          .subscribe(o -> {
+            runOnUiThread(() -> {
```

Comment:
The call to `applySchedulres` above sets observation to happen on the main thread which makes this call redundant.